torch/testing/_internal/common_methods_invocations.py

```diff
@@ -651,6 +651,21 @@ def sample_inputs_amax_amin(op_info, device, dtype, requires_grad):
                              args=args)
                  for size, args in test_cases)
 
+def sample_inputs_argmax_argmin(op_info, device, dtype, requires_grad):
+    test_cases = (
```

Comment:
For a 1D empty tensor & the empty tensor(S, 0, S), invoking `argmax` produces a Runtime error:

> RuntimeError: cannot perform reduction function argmax on a tensor with no elements because the operation does not have an identity

This behavior is consistent with numpy

Reply:
This looks great, thanks for improving the test coverage.